src/main/webapp/script.js

```diff
@@ -12,7 +12,105 @@
+    this.expertise = expertise;
+}
 
+/*
+Function: addTableCards()
+Adapts the feed to the username whether is a mentor or mentee to show the right information
+    
+Notes:
+    Up until now works in a local way with structs containing dummy info 
+    Next step is to adapt it to work with datastorage
+*/
+function addTableCards() {
 
+    /*Could be an option to obtain the value from the redirection from other pages, still in progress...
+    const queryString = window.location.search;
+    const urlParams = new URLSearchParams(queryString);
+    const username = urlParams.get('username');*/
 
+    const username = "john123";
+
+    var myMentees = [
+        new dummyDBMentee('John S', 'john@myemail.com', 'john123', 'mySchool', 2021, "I'm a student", "I like to learn", "C++"),
+        new dummyDBMentee('Mary S', 'mary@myemail.com', 'mary123', 'mySchool', 2022, "I'm a student", "I like to learn", "Java"),
+        new dummyDBMentee('Nick S', 'nick@myemail.com', 'nick123', 'mySchool', 2024, "I'm a student", "I like to learn", "Go")
+    ];
+
+    var myMentors = [
+        new dummyDBMentor('John T', 'john@myemail.com', 'john124', 'Senior Software Engineer', 'MyCollege', "I'm a software engineer", "JavaScript"),
+        new dummyDBMentor('Mary T', 'mary@myemail.com', 'mary124', 'UX Designer', 'MyCollege', "I'm a UX specialist", "Accesibility")
+    ];
+
+    //Having a mentor username, showing list of mentees
+    for(var i=0; i<myMentors.length; i++){
+        if(username == myMentors[i].username){
+            for(var j=0; j<myMentees.length; j++){
+                var tag = document.createElement('div');
+                tag.innerHTML = '<div class="card">' + 
+                    '<div id="card-img"><img src="images/img1.png"></img></div>' + 
+                    '<div id="card-nameholder"><h3>' + myMentees[j].name + '</h3></div>' + 
+                    '<div id="card-seemore-button">' + 
+                        '<button type="button" class="btn btn-info" data-toggle="collapse" data-target="#more-info' + j + '">See more</button>' + 
+                    '</div>' + 
+                    '<div id="more-info' + j + '" class="collapse" style="padding: 2px 120px;">' + 
+                        '<strong>E-mail: </strong><p>' + myMentees[j].email + '</p>' + 
+                        '<strong>School: </strong><p>' + myMentees[j].school + '</p>' + 
+                        '<strong>Class Standing: </strong><p>' + myMentees[j].classStanding + '</p>' + 
+                        '<strong>About me: </strong><p>' + myMentees[j].intro + '</p>' + 
+                        '<strong>My objectives: </strong><p>' + myMentees[j].reason + '</p>' + 
+                        '<strong>Skills Needed: </strong><p>' + myMentees[j].desiredSkills + '</p>' + 
+                    '</div>' + '</div>';
+                
+                var element = document.getElementById("content");
+                element.appendChild(tag);
+            }
+            break;
```

Comment:
Why do you have this break here? (also below)

Reply:
I was trying to stop the for loop in case it found the username, to avoid going through all the data.